sqlite/db.go

```diff
@@ -0,0 +1,111 @@
+package sqlite
+
+import (
+	"database/sql"
+	"fmt"
+	"os"
+
+	sqlite "github.com/mutecomm/go-sqlcipher" // We require go sqlcipher that overrides default implementation
+)
+
+// The default number of kdf iterations in sqlcipher (from version 3.0.0)
+// https://github.com/sqlcipher/sqlcipher/blob/fda4c68bb474da7e955be07a2b807bda1bb19bd2/CHANGELOG.md#300---2013-11-05
+// https://www.zetetic.net/sqlcipher/sqlcipher-api/#kdf_iter
+const defaultKdfIterationsNumber = 64000
+
+// The reduced number of kdf iterations (for performance reasons) which is
+// currently used for derivation of the database key
+// https://github.com/status-im/status-go/pull/1343
+// https://notes.status.im/i8Y_l7ccTiOYq09HVgoFwA
+const kdfIterationsNumber = 3200
+
+const exportDB = "SELECT sqlcipher_export('newdb')"
+
+// EncryptDatabase encrypts an unencrypted database with key
+func EncryptDatabase(oldPath string, newPath string, key string) error {
```

Comment:
You probably don't need this `fn` I think it's only used by us as we had to migrate

Reply:
right, i forgot to remove it